en/upgrade-tidb-operator.md

```diff
@@ -40,6 +40,23 @@ This document describes how to upgrade TiDB Operator and Kubernetes.
     helm upgrade tidb-operator pingcap/tidb-operator --version=${version} -f ${HOME}/tidb-operator/${version}/values-tidb-operator.yaml
     ```
 
+    After all of pods start normally, execute:
+
+    ```shell
+    kubectl get po -n tidb-admin -l app.kubernetes.io/instance=tidb-operator -o yaml | grep image:
+    ```
+
+    Tidb operator would be upgraded successfully if outputs look like below, `${version}` represents the wanted version of tidb operator.
+
+    ```
+      image: pingcap/tidb-operator:${version}
+      image: docker.io/pingcap/tidb-operator:${version}
+      image: pingcap/tidb-operator:${version}
+      image: k8s.gcr.io/kube-scheduler:v1.14.10
+      image: k8s.gcr.io/kube-scheduler:v1.14.10
+      image: docker.io/pingcap/tidb-operator:${version}
```

Comment:
```suggestion
    image: pingcap/tidb-operator:${version}
    image: docker.io/pingcap/tidb-operator:${version}
    image: pingcap/tidb-operator:${version}
    image: k8s.gcr.io/kube-scheduler:v1.14.10
    image: k8s.gcr.io/kube-scheduler:v1.14.10
    image: docker.io/pingcap/tidb-operator:${version}
```

Reply:
The version of `kube-scheduler` should also be a variable.